pkg/cmd/pulumi/new.go

```diff
@@ -508,8 +507,10 @@ func getStack(stack string, opts display.Options) (backend.Stack, string, string
 	name := ""
 	description := ""
 	if s != nil {
-		if cs, ok := s.(httpstate.Stack); ok {
-			tags := cs.Tags()
+		tags, err := s.Tags()
```

Comment:
Could we check for that error and propagate all others. If a new error is added, or something unexpected happens, I don't want to silently eat errors. 

If the only possible error for `Tags` is "not supported", maybe change the return type from `error` to a bool indicating if tags are supported.